### DISCUSSION TITLE
Snake

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#26] [Games]
# Snake

https://wasm4.org/play/snake/?giscus=145c74b4d021b9330f43bae1bIGGFPxroAenIJR%2FcnMxSxL26MgrrU4dabny1NkaIwAJe7KyQWXkn%2FMKlu%2F44RXv1z%2B21ePLga9chKVjf2nIaZH8otI2usthvTOggAPoLJkGgygUA12TrwYZq4s%3D

**Comment by ChengDawei7** [comment]
This game has a major flaw of not being able to handle fast inputs and making the snake die going into itself. 

Say the snake is going down and you press Left and Up. If you press these two buttons too quickly the snake only goes up into itself and dies. Really frustrating!
